generate_xml_from_google_services_json.py

```diff
@@ -108,6 +108,33 @@ def construct_plist_dictionary(xml_root):
   return plist_dict
 
 
+def update_dict_keys(key_map, input_dict):
+  """Creates a dict from input_dict with the same values but new keys.
+
+  Two dictionaries are passed to this function: the key_map that represents a
+  mapping of source keys to destination keys, and the input_dict that is the
+  dictionary that is to duplicated, replacing any key that matches a source key
+  with a destination key. Source keys that are not present in the input_dict
+  will not have their destination key represented in the result.
+
+  In other words, if key_map is `{'old': 'new', 'foo': 'bar'}`, and input_dict
+  is `{'old': 10}`, the result will be `{'new': 10}`.
+
+  Args:
+    key_map: A dictionary of source keys to destination keys.
```

Comment:
Though it is obvious here but it is always a good idea to put the argument type in the docstring.
Also, any reason why we want to call one thing a map and other a dictionary?

Reply:
I don't have a good reason for the names, other than it sounded natural. The first argument maps old keys to new keys, the other is just some arbitrary dictionary. I could call it `key_dict` or `key_map_dict` if you prefer, or rename `input_dict` to just `input` or something. 

Naming is hard.

Reply:
Yup, this one is tricky. I think it does sound natural. Let's leave it as it is.